temba/msgs/models.py

```diff
@@ -881,7 +881,7 @@ def process_message(cls, msg):
 
         # Sending data to Chatbase API
         (chatbase_api_key, chatbase_version) = msg.org.get_chatbase_credentials()
-        if chatbase_api_key:
+        if chatbase_api_key and msg.channel:
```

Comment:
I think better to be more explicit and check `msg.contact.is_test` before even calling `msg.org.get_chatbase_credentials()`